xclim/indices/_agro.py

```diff
@@ -341,3 +344,71 @@ def water_budget(
 
     out.attrs["units"] = pr.attrs["units"]
     return out
+
+
+@declare_units(pr="[precipitation]", thresh="[precipitation]")
+def rolling_drydays_events(
+    pr: xarray.DataArray, thresh: str = "1.0 mm/day", window: int = 3, freq: str = "YS"
+) -> xarray.DataArray:
+    """
+    Return the number of periods of minimum X days during which the accumulated precipitation is under the
+    threshold, for each resampling period.
+
+    Parameters
+    ----------
+    pr : xarray.DataArray
+      Daily precipitation.
+    thresh : str
+      Precipitation value under which a period is considered dry.
+    window : int
+      Minimum number of days where the accumulated precipitation is under threshold.
+    freq : str
+      Resampling frequency.
+
+    Returns
+    -------
+    xarray.DataArray
+      The number of periods of minimum {window} days for each resampling period.
+    """
+    thresh = convert_units_to(thresh, pr, "hydro")
+
+    out = (
+        (pr.rolling(time=window, center=True).sum() < thresh)
```

Comment:
if we follow @aulemahal suggestion and decide to use `xc.core.units.rate2amount` should this become a rolling mean?

Reply:
We still want the accumulated precipitation amount over the whole period, so we need to keep the rolling sum.

Reply:
Running the code/tests now & I see how this works